frontend/src/types/types.ts

```diff
@@ -11,6 +11,7 @@ export interface CommitResult extends CommitResultRaw {
   repoId: string;
```

Comment:
These seem to be be a duplicate of #1937. I think for new pull requests linked to #1936, it is recommended to follow the sequential merge strategy so as to not include the duplicate upstream changes from previous pull requests. Otherwise, if implementation changes from the upstream pull request, the changes need to be made here as well. You can keep these changes for this pull request since the previous pull request is about to be merged

Reply:
Yup will try to follow this in the future - I had to include the changes from #1937 in this case as I needed the types from `store`. Will try to not include duplicate changes in future PRs unless absolutely necessary